project/frontend/src/App.js

```diff
@@ -241,7 +164,11 @@ function App() {
             innerNav={compId != 0}
             compIdChanged={setCompId}
           ></HeaderBar>
-          <Layout>{pageRoute}</Layout>
+          {!authStateReceived ? (
+            <div>Signing in...</div>
+          ) : (
+            <Layout>{pageRoute}</Layout>
+          )}
```

Comment:
Continuation of above comment - this could be replaced with just a `{pageContent}`

Reply:
Done.